### PR TITLE
Add cooldown to simple_animal Mecha attacks

### PR DESCRIPTION
**Pull request by Markolie**
Fixes https://github.com/ParadiseSS13/Paradise/issues/6122.

🆑 Markolie
fix: simple_animals now have an attack cooldown when attacking mechas.
/🆑

### CODE REVIEW

File: code/game/mecha/mecha.dm [hunk 1]
```diff
@@ -501,6 +501,8 @@
 
 /obj/mecha/attack_animal(mob/living/simple_animal/user as mob)
 	log_message("Attack by simple animal. Attacker - [user].",1)
+	user.changeNext_move(CLICK_CD_MELEE)
+	user.do_attack_animation(src)
```
Comment (Crazylemon64) [inline code/game/mecha/mecha.dm]: Should the piece of code in `simple_animal` that calls `attack_animal` have this code instead, or is there something I am missing?

Comment (Markolie) [inline code/game/mecha/mecha.dm]: I don't believe so, we generally have to add attack cooldowns to every attack_animal proc separately. The other attack_ procs in the mecha file also have custom cooldowns (attack_alien, etc.)

Comment (Crazylemon64) [inline code/game/mecha/mecha.dm]: https://github.com/ParadiseSS13/Paradise/blob/master/code/_onclick/other_mobs.dm#L46-L47
This is what calls `attack_animal` in most cases afaik

**Comment by Crazylemon64** [comment]
travis messed up for some reason - the boxstation build didn't work. 1 sec

**Comment by Crazylemon64** [comment]
wat

**Pull request merged**